packages/neuron-wallet/src/controllers/wallets/index.ts

```diff
@@ -72,6 +72,7 @@ export default class WalletsController {
       name,
       password,
       mnemonic,
+      isImport: true,
```

Comment:
```suggestion
      isImporting: true,
```

Reply:
`isImport` means this wallet is generate by import or create,  I'm not sure `isImporting` is better to describe this ?

Reply:
`isImport` means this wallet is generate by import or create,  I'm not sure `isImporting` is better to describe this ?

Reply:
Then perhaps `isImported` is more appropriate.

Reply:
`isImported` is more like the `import` action done or not, what do you think?

Reply:
Renamed to `isImporting`